HW4-KZaballa.py

```diff
@@ -22,8 +24,20 @@
 # regression-in-python-3d3f8f06bf8
 
 
-def boolean_processing(predictor, response):
+def boolean_processing(predict_df, response):
     # fill in with logistic regression
+    for idx, column in enumerate(predict_df.T):
+        if (np.dtype(predict_df[idx]) == str) or (np.dtype(predict_df[idx]) == bool):
+            print("This variable is categorical")
+        elif (np.dtype(predict_df[idx]) == int) or (np.dtype(predict_df[idx]) == float):
+            print("This variable is continuous")
+        else:
+            print("This variable is ", np.dtype(predict_df[idx]))
+    logreg = LogisticRegression()
+    rfe = RFE(logreg, predict_df.columns.size)
+    rfe = rfe.fit(predict_df, response.values.ravel())
+    print(rfe.support_)
+    print(rfe.ranking_)
     return print("boolean processing")
 
 
```

Comment:
Im not sure what line 81 is starting but comments in that section and below it would be nice so that there is more description to the reader for what is being done.